maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -304,8 +322,35 @@ def _update_pm_workload(self):
                 vm = self._live_vms[vm_id]
                 total_pm_cpu_cores_used += vm.cpu_utilization * vm.cpu_cores_requirement
             pm.update_cpu_utilization(vm=None, cpu_utilization=total_pm_cpu_cores_used / pm.cpu_cores_capacity)
+            if pm.cpu_utilization > 100:
```

Comment:
how about？ or which one do you prefer?

no matter what the overload handling option is

the maximum consumption or calculated according to the formula?